src/pages/console/authz/permissions.mdx

```diff
@@ -44,4 +44,4 @@ If you want your data model to be publicly accessible, switch to API_KEY or IAM
 1. Using the *Books* data model that you created in the [Create a data model example](/console/data/data-model#Create-a-data-model-example), set the authorization mode to **API Key**. 
 2. In the **Model** pane on the right, expand the **Anyone** window. Choose **Read** to specify that any signed in user has read access to the data in the *Book* model. The settings look as follows.
 
-![GSA](/images/console/7_publicauthreadonly.png)
\ No newline at end of file
+![Public authorization rules on a GraphQL data model](/images/console/7_publicauthreadonly.png)
```

Comment:
Same here regarding the "The settings look as follows", is there any extra info the photo not being explained above? If so then we should write it out in the text above or explain it some more in the alt text.